src/components/Button/Button.js

```diff
@@ -49,9 +48,13 @@ const ButtonBase = styled.button`
 `;
 
 const Button = ({
-  variant, outline, className, theme: propTheme, ...props
+  variant, outline, className, theme: propTheme, size, ...props
 }) => {
-  const buttonColorPalette = propTheme.colors.variants[variant] || get('colors.variants.primary');
+  const buttonSizingTypography = propTheme.buttons.sizingAndTypography;
+  const sizingAndTypography = buttonSizingTypography[size] || buttonSizingTypography.medium;
```

Comment:
do u know if you could use a variant for this? like https://github.com/hack4impact-uiuc/bridge/blob/master/src/components/Text/Text.js

but with `buttons.sizingAndTypography`

Reply:
yup changed to variant